cmd/edgectl/aes_install_errors.go

```diff
@@ -120,31 +124,29 @@ func (i *Installer) IncompatibleCRDVersionsError(installedVersion string) Result
 This tool does not support upgrades/downgrades at this time.
 The installer will now quit to avoid corrupting an existing installation of AES.
 `
-	message := fmt.Sprintf("existing AES %s found when installing AES %s\n%s",
-		installedVersion, i.version, abortExisting)
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/incompatible-crd-versions"
+	message := fmt.Sprintf("Visit %v for a more detailed explanation and step-by-step instructions about updating CRD verions to continue installing Ambassador Edge Stack\n%s",
```

Comment:
Not sure about the trailing "n%s" here?

Reply:
It'll append the `abortExisting` paragraph, although I agree it could be simplified and merged.